src/js/templates/maps/layer-category-item.html

```diff
@@ -0,0 +1,15 @@
+<div class="<%= classNames.baseClass %>">
```

Comment:
Okay so it looks like all views are wrapped in a div and given a class name of whatever the className attribute of your view is, so specifying a className attribute should be enough, and then you don't need to have the extra div (right now I think your DOM looks like two divs with identical classes?)

Reply:
OH good catch!